zerver/tests/test_openapi.py

```diff
@@ -929,12 +930,29 @@ def test_attributes(self) -> None:
             if path in EXCLUDE:
                 continue
             for method, operation in path_item.items():
-                # Check if every file has an operationId
+                # Check if every file has an operationId.
                 assert "operationId" in operation
```

Comment:
Probably better still to just delete this comment :) — it's not saying anything the line after it doesn't already clearly say.

(Except its use of the term "file"; but I think that's just an inaccurate choice of words and not chosen to convey some subtle meaning.)

Reply:
Yeah, this might have been a comment from when there wasn't a shared template for the endpoints documented in `zulip.yaml`. Removed it in the updated changes.